### ISSUE TITLE
Optimus prime Invalid choice Aborting

### ISSUE THREAD

**Issue by HeatSeaker** [Askannz/optimus-manager#480]
I'm running Arch, on my HP notebook. Specs;
Processor: Intel Core i5-8265U @1.60GHz 8
iGPU: Mesa Intel UHD Graphics 620 (WHL GT2)
dGPU: Nvidia Geforce MX110 (running proprietary graphics drivers)
Session type = x11
I was using gdm from pacman before. I uninstalled gdm and libgdm (using `-Rcsu` options with pacman before installing gdm-prime from AUR)
I have installed gdm-prime from AUR (using paru)
Output of `systemctl status optimus-manager.service` is: https://i.imgur.com/7iHPGut.png

When I try to run `optimus-prime --switch nvidia`
it says `Invalid choice. Aborting`
Screenshot: https://i.imgur.com/QKPtceQ.png

There are no log files created in the `var/log/optimus-manager/switch` when I run the switch command
Though there are other log files (I think they are being created when I restart the laptop/session?) the latest's contents are: https://i.imgur.com/2yPqci1.png

Please let me know if further details are required.
It's my first time setting up dual GPU (and am fairly new to arch itself) so please pardon me if I am missing something really obvious.

**Comment by Askannz** [comment]
That is strange. Does it skip to "Invalid choice" without you inputting anything?

**Comment by HeatSeaker** [comment]
Sorry for getting back this late. If I recall correctly the only thing I put in was what profile I want it to switch to. Other than that, it proceeded with that error

**Comment by Askannz** [comment]
Oh that's why. Only y,Y,n,N are considered valid choices, not "Enter". But I agree that's a bit counter to the usual behavior of CLI apps, so probably worth changing that behavior so that it picks a default instead.

In the meantime, just write "y" or pass ` --no-confirm` to skip the prompt.

**Comment by HeatSeaker** [comment]
Alright, I will try that and leave feedback if that works, Thanks :D

**Comment by HeatSeaker** [comment]
Sorry for the very late feedback. Yes, the switching works just fine. Dumb of me to not think that the error wasn't about the mode but about what I was inputting when asked for confirmation. Now the issue is that I get a black screen in `integrated` mode (that is not even lit, like if the laptop is turned off) But I believe it's completely unrelated to this, as `nvidia` mode is working just fine. I'll go and troubleshoot that and post my results here.

**Comment by HeatSeaker** [comment]
Alright, today I booted into my laptop. And `integrated` mode is also working fine. Yesterday, no matter how many times I rebooted, it did not fix (before trying the switch to `nvidia` mode). Was using the `nvidia` mode for some time and I left my laptop on and it died due to battery running out. Today when I booted it again, the `X.org` session is rendering even on the `integrated` mode with the session type `X11`
